Windows build is failing;

```
cl.exe -O2 -Wall -std=c99 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64   -c -o flash.o flash.c
Microsoft (R) C/C++ Optimizing Compiler Version 19.40.33811 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

cl : Command line warning D9035 : option 'o' has been deprecated and will be removed in a future release
cl : Command line warning D9002 : ignoring unknown option '-std=c99'
flash.c
flash.c(31): fatal error C1083: Cannot open include file: 'getopt.h': No such file or directory
mingw32-make: *** [flash.o] Error 2
<builtin>: recipe for target 'flash.o' failed
```

--- a/.ci_support/win_64_.yaml
+++ b/.ci_support/win_64_.yaml
@@ -1,10 +1,10 @@
-c_stdlib:
-- vs
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 m2w64_c_compiler:
+- m2w64-toolchain
+m2w64_c_stdlib:
 - m2w64-toolchain
 target_platform:
 - win-64

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -15,10 +15,10 @@ build:
 requirements:
   build:
     - make
-    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('c') }}        # [unix]
+    - {{ stdlib('c') }}          # [unix]
     - {{ compiler('m2w64_c') }}  # [win]
-    - m2w64-toolchain  # [win]
-    - {{ stdlib("c") }}
+    - {{ stdlib('m2w64_c') }}    # [win]
   host:
     - zlib
   run:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -17,6 +17,7 @@ requirements:
     - make
     - {{ compiler('c') }}  # [unix]
     - {{ compiler('m2w64_c') }}  # [win]
+    - m2w64-toolchain  # [win]
     - {{ stdlib("c") }}
   host:
     - zlib

